src/appleseed/foundation/string/string.h

```diff
@@ -896,6 +896,11 @@ inline std::string replace_special_xml_characters(const std::string& s)
     return result;
 }
 
+inline std::string replace_special_xml_characters(const char* s){
```

Comment:
This function isn't needed as a `const char*` is implictly convertible to a `std::string`, hence the original function can be used without issues.

Reply:
Okay, I'll check.